src/Effect.tsx

```diff
@@ -63,7 +63,7 @@ export function Effect(
       if (onActivation) {
         onActivation(node);
       }
-      document.addEventListener('keyup', onKeyPress);
+      document.addEventListener('keydown', onKeyDown);
       document.addEventListener('click', onClick);
```

Comment:
Look like this also has to be updated. However "click outside" is not defined for "native modals" (they just ignore it)